packages/tabs/stories/Tabs.stories.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Story, Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Tabs, TabList, Tab, TabPanel } from '@zendeskgarden/react-tabs';
+
+export default {
+  title: 'Components/Tabs',
+  subcomponents: { Tabs, TabList, Tab, TabPanel },
```

Comment:
By only listing `subcomponents` we don't have to override the arguments that we don't want to display in the story controls.